apps/web-giddh/src/app/ledger/components/ledger-columnar-report-table/ledger.columnar.report.table.component.ts

```diff
@@ -176,4 +181,16 @@ export class LedgerColumnarReportTableComponent implements OnInit, OnDestroy, On
 
     }
 
+    /**
+     * Tracks the entry by Entry ID
+     *
+     * @param {*} index Index of current entry
+     * @param {*} entry Entry data
+     * @return {*}  {string} Entry ID
+     * @memberof LedgerColumnarReportTableComponent
+     */
+    public trackByEntryId(index: any, entry: any): string {
```

Comment:
will string return type work if entry is null/undefined?

Reply:
Entry will not be null/undefined as we are looping over the values provided by the API